asttokens/util.py

```diff
@@ -320,11 +325,11 @@ def get(self, obj, cls):
   # Python 2 doesn't support non-ASCII identifiers, and making the real patched_generate_tokens support Python 2
   # means working with raw tuples instead of tokenize.TokenInfo namedtuples.
   def patched_generate_tokens(original_tokens):
-    # type: (Any) -> Any
-    return original_tokens
+    # type: (Iterable[TokenInfo]) -> Iterator[TokenInfo]
+    return iter(original_tokens)
 else:
   def patched_generate_tokens(original_tokens):
-    # type: (Iterator[tokenize.TokenInfo]) -> Iterator[tokenize.TokenInfo]
+    # type: (Iterable[tokenize.TokenInfo]) -> Iterator[tokenize.TokenInfo]
```

Comment:
```suggestion
    # type: (Iterable[TokenInfo]) -> Iterator[TokenInfo]
```

just for consistency